tools/ci_build/build.py

```diff
@@ -120,7 +120,7 @@ def parse_arguments():
     parser.add_argument("--skip_submodule_sync", action='store_true', help="Don't do a 'git submodule update'. Makes the Update phase faster.")
 
     parser.add_argument("--use_jemalloc", action='store_true', help="Use jemalloc.")
-    parser.add_argument("--use_mimalloc", action='store_true', help="Use mimalloc.")
+    parser.add_argument("--use_mimalloc", action='store_false', help="Use mimalloc.")
     parser.add_argument("--use_openblas", action='store_true', help="Build with OpenBLAS.")
```

Comment:
Is this PR still a WIP? The comments about "explicitly disable subbing mimalloc in for changes that interact with python", and this change for led me to think so. Is that the case?

Reply:
No this PR isn't still a WIP - this is adjacent to the referenced change. As far as I'm aware the plan is to eventually turn on mimalloc as a swap in for the python allocator on windows - but more performance testing needs to happen for that switch to be flipped on by default since that could have a big impact on workloads. I've been working with @ybrnathan - he'll have more context.

The existing switch/work itself has been off by default this entire time, but this particular mimalloc change is targeted enough to warrant turning on without more performance tuning.